YCAI/src/components/dashboard/LinkAccount.tsx

```diff
@@ -90,11 +102,11 @@ export const LinkAccount: React.FC<LinkAccountProps> = ({ auth }) => {
 
   const handleChannelSubmit: React.MouseEventHandler<HTMLButtonElement> =
     async () => {
-      if (inputRef.current?.firstChild !== null) {
-        const channelId = (inputRef.current?.firstChild as any).value;
-        await registerCreatorChannel(channelId, {
+      if (channel.length > 0) {
```

Comment:
we can make this like 5 at least? I don't think there's shorter yt channel ids

Reply:
or we can move this check to the button, which is disabled till the `channel.length < 5`

Reply:
moved to the button